modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -94,19 +94,23 @@ const attachUiMotherships = (editor: Editor, uiRoot: SugarElement<HTMLElement |
   Attachment.attachSystem(uiRoot, uiRefs.dialogUi.mothership);
 };
 
-const render = async (editor: Editor, uiRefs: ReadyUiReferences, rawUiConfig: RenderUiConfig, backstage: UiFactoryBackstage, args: RenderArgs): Promise<ModeRenderInfo> => {
+const render = (editor: Editor, uiRefs: ReadyUiReferences, rawUiConfig: RenderUiConfig, backstage: UiFactoryBackstage, args: RenderArgs): ModeRenderInfo => {
   const { mainUi, uiMotherships } = uiRefs;
   const lastToolbarWidth = Cell(0);
   const outerContainer = mainUi.outerContainer;
 
-  await loadIframeSkin(editor);
+  loadIframeSkin(editor);
 
   const eTargetNode = SugarElement.fromDom(args.targetNode);
   const uiRoot = SugarShadowDom.getContentContainer(SugarShadowDom.getRootNode(eTargetNode));
 
   Attachment.attachSystemAfter(eTargetNode, mainUi.mothership);
   attachUiMotherships(editor, uiRoot, uiRefs);
 
+  editor.on('SkinLoaded', () => {
+    setToolbar(editor, uiRefs, rawUiConfig, backstage);
+  });
```

Comment:
Maybe put this before the line 102 `loadIframeSkin(editor)`

Reply:
done